### ISSUE TITLE
Charts overlap

### ISSUE THREAD

**Issue by patricktokeeffe** [node-red/node-red-dashboard#57]
I'm observing chart groups overlapping (screenshot below). The overlap wrapping is consistent and occurs in several browsers (Chrome/Win7, Chromium/Linux, Firefox/Win7 & Lightning/Android). 

I had thought the cause was mixing automatic with explicit plot sizes (groups with explicitly sized plots would end up underneath), but further experimentation shows it's not that simple.

Here's what I'm seeing (@100% zoom):

![overlapped_charts_100pct](https://cloud.githubusercontent.com/assets/1348834/18449436/a0d78854-78db-11e6-8851-e3d206be304b.png)

When I zoom in, then back out to 90% I get something like this:

![overlapped_charts_90pct](https://cloud.githubusercontent.com/assets/1348834/18449461/b981fe8e-78db-11e6-98a5-9d79d3fd3a05.png)

And at 75% but with reduced page width, the stacking looks like this:

![overlapped_charts_75pct_narrow](https://cloud.githubusercontent.com/assets/1348834/18449517/f3419b2a-78db-11e6-8030-e238c20addf1.png)

Here's the order of my groups within the dashboard tab editor; unless specified, the plots all use auto size and all groups use default width.

* *...some other tabs...*
* *Outdoor Rack*
  * TSI DustTrak II
  * Dylos DC1100 (6x5)
  * T300U
  * T400
  * Licor LI840A (6x4 and 6x2)
  * T200U

> *Edit: I forgot to specify which groups have set-size plots.*

**Comment by dceejay** [comment]
Hi Patrick - is this version 2.0.2 of the dashboard ?

**Comment by patricktokeeffe** [comment]
No, it was not but I updated to 2.0.2 and still see this behavior. 

----

On a separate note, upgrading apparently nuked my flows: I had to make a fake change and redeploy all flows from an existing browser session in order for them to show up again in a new browser session (good thing I didn't close the original browser tab!). My dashboard tabs & groups also appear to be hosed up... (still working on that issue report).
